src/icon-button/icon-button.style.tsx

```diff
@@ -1,12 +1,17 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import { Color } from "../color/color";
 
-export const Main = styled.button`
+interface StyleProps {
+    $outline: "none" | "browser";
+    $highlight: boolean;
+}
+
+export const Main = styled.button<StyleProps>`
     display: flex;
     justify-content: center;
     align-items: center;
     padding: 1.5rem;
-    background-color: ${Color.Neutral[8]};
+    background-color: transparent;
```

Comment:
almost every component was overriding `background-color` from white to transparent, so set transparent as the default